tasks/less.js

```diff
@@ -169,10 +169,14 @@ module.exports = function(grunt) {
 
   var parseVariableOptions = function(options) {
     var pairs = _.toPairs(options);
-    var output = '';
-    pairs.forEach(function(pair) {
-      output += '@' + pair[0] + ':' + pair[1] + ';';
-    });
+    var output = _.reduce(pairs, function(accumulator, pair) {
+      var variableName = pair[0];
+      var variableValue = pair[1];
+      if (variableValue !== undefined && variableValue !== null) {
```

Comment:
I've updated the code using `typeof`.

I'd like to keep the `null` check because:
- it is handy to avoid all non-js value  to be transformed to some less value 
- and in addition to that it will also make devs life easier, no need to cleanup the object passed in from null values

Do you agree?